packages/datagrid/package.json

```diff
@@ -32,37 +32,37 @@
     "url": "https://github.com/Talend/ui.git"
   },
   "dependencies": {
-    "ag-grid-community": "^24.0.0",
-    "ag-grid-react": "^24.0.0",
-    "classnames": "^2.2.5",
+    "ag-grid-community": "^25.1.0",
+    "ag-grid-react": "^25.1.0",
+    "classnames": "^2.3.1",
```

Comment:
I prefere avoid this kind of upgrade which creates lots of duplicate in our node_modules.
i will apply a global yarn ugprade to the repository to realign dependencies